src/features/events/components/EventOverviewCard/EventParticipantsCard.tsx

```diff
@@ -0,0 +1,221 @@
+import {
+  Box,
+  ClickAwayListener,
+  Divider,
+  Link,
+  Paper,
+  Popper,
+  TextField,
+  Typography,
+} from '@mui/material';
+import { FC, useState } from 'react';
+
+import EventDataModel from 'features/events/models/EventDataModel';
+import messageIds from 'features/events/l10n/messageIds';
+import NextLink from 'next/link';
+import { Settings } from '@mui/icons-material';
+import theme from 'theme';
+import { useMessages } from 'core/i18n';
+import ZUICard from 'zui/ZUICard';
+import ZUINumberChip from 'zui/ZUINumberChip';
+import ZUIPerson from 'zui/ZUIPerson';
+
+type EventParticipantsCardProps = {
+  campId: string;
+  model: EventDataModel;
+};
+
+const EventParticipantsCard: FC<EventParticipantsCardProps> = ({
+  model,
+  campId,
+}) => {
+  const eventData = model.getData().data;
+  const messages = useMessages(messageIds);
+  const reqParticipants = eventData?.num_participants_required ?? 0;
+  const availParticipants = eventData?.num_participants_available ?? 0;
+
+  const [newReqParticipants, setNewReqParticipants] = useState<number | null>(
+    reqParticipants
+  );
+  const [anchorEl, setAnchorEl] = useState<
+    null | (EventTarget & SVGSVGElement)
+  >(null);
+
+  if (!eventData) {
+    return null;
+  }
+
+  const getParticipantStatus = () => {
+    const diff = reqParticipants - availParticipants;
+
+    if (diff <= 0) {
+      return theme.palette.statusColors.green;
+    } else if (diff === 1) {
+      return theme.palette.statusColors.orange;
+    } else {
+      return theme.palette.statusColors.red;
+    }
+  };
+
+  return (
+    <Box>
+      <ZUICard
+        header={messages.eventParticipantsCard.header()}
+        status={
+          <Box display="flex">
+            <ZUINumberChip
+              color={getParticipantStatus()}
+              outlined={true}
+              size="sm"
+              value={`${availParticipants}/${reqParticipants}`}
+            />
+            <Box ml={1}>
+              <Settings
+                color="secondary"
+                cursor="pointer"
+                onClick={(event) =>
+                  setAnchorEl(anchorEl ? null : event.currentTarget)
+                }
```

Comment:
![bild](https://user-images.githubusercontent.com/58265097/230387793-88b44150-7ad7-4f64-aa3e-4be162e9e32e.png)
When playing around I have gotten this to happen two times, but cannot consistently recreate it so i have not examined why it happens. But my thought was that it could be that th anchor element is somehow not always set, and it anchors to the top left instead?

Reply:
I have not been able to recreate this, so am not able to do anything about it currently. Your analysis seems correct, but I am not sure what could cause that. Perhaps leave this and create an issue for it if we have no clue how it is happening.

Reply:
very good idea! :)